edx_arch_experiments/kafka_consumer/tests/test_bridge_use.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+from django.test import TestCase
+
+from confluent_kafka.schema_registry import SchemaRegistryClient
+from confluent_kafka.schema_registry.avro import AvroSerializer, AvroDeserializer
+from opaque_keys.edx.keys import CourseKey
+from openedx_events.bridge.avro_attrs_bridge import AvroAttrsBridgeKafkaWrapper
+from openedx_events.learning.signals import COURSE_ENROLLMENT_CREATED
+from openedx_events.learning.data import CourseEnrollmentData, CourseData, UserPersonalData
+from unittest.mock import Mock
+
+class BridgeTest(TestCase):
+
+    def test_produce_and_consume_event_with_bridge(self):
+        my_bridge = AvroAttrsBridgeKafkaWrapper(COURSE_ENROLLMENT_CREATED)
```

Comment:
An instance of a bridge is created from a specific signal, which is an instance of OpenEdxPublicSignal